src/metaswitch/sasclient/sender.py

```diff
@@ -118,12 +119,17 @@ def connect(self):
                 self._connected = True
 
     def disconnect(self):
-        logger.info("Disconnecting")
+        logger.debug("Disconnecting")
         # It's possible that the socket doesn't even exist yet, so we have nothing to do.
         if self._sas_sock is None:
             return
-        self._sas_sock.shutdown(socket.SHUT_RDWR)
-        self._sas_sock.close()
+        try:
+            self._sas_sock.shutdown(socket.SHUT_RDWR)
+            self._sas_sock.close()
+        except Exception:
```

Comment:
Just to be paranoid, are there any other exceptions that can be thrown here other than those caused by the connection being down? We don't want to end up leaking connections or something. Perhaps it's worth at least including the Exception error message in the debug log?